integrations/mistral/src/haystack_integrations/components/generators/mistral/chat/chat_generator.py

```diff
@@ -20,9 +20,21 @@ class MistralChatGenerator(OpenAIChatGenerator):
     directly to this component via the `**generation_kwargs` parameter in __init__ or the `**generation_kwargs`
     parameter in `run` method.
 
+    Key Features and Compatibility:
+    - **Primary Compatibility**: Designed to work seamlessly with the Mistral API Chat Completion endpoint.
+    - **Streaming Support**: Supports streaming responses from the Mistral API Chat Completion endpoint.
+    - **Customizability**: Supports all parameters supported by the Mistral API Chat Completion endpoint.
+
+    This component uses the ChatMessage format for structuring both input and output,
+    ensuring coherent and contextually relevant responses in chat-based text generation scenarios.
+    Details on the ChatMessage format can be found in the
+    [OpenAI docs](https://github.com/openai/openai-python/blob/main/chatml.md)
```

Comment:
This URL is outdated (was previously deprecated by OpenAI).
I would refer to our docs: https://docs.haystack.deepset.ai/v2.0/docs/data-classes#chatmessage